guides/doc-Administering_Red_Hat_Satellite/topics/log-files-provided-by-satellite.adoc

```diff
@@ -4,6 +4,11 @@
 
 {ProjectName} provides system information in the form of notifications and log files.
 
+ifndef::satellite[]
+Note that depending on your specific configuration, your log directories vary.
+For example, if you do not use the Katello plugin, some of the entries in this table are not present in your deployment. 
+endif::[]
```

Comment:
I'd prefer to:
* hide `candlepin` and `tomcat` on non-Katello
* `/var/log/rhsm` should be hidden on `foreman-deb`
* `/var/log/httpd` should be `/var/log/apach2` on `foreman-deb`
* `/var/log/messages` should be `/var/log/daemon.log` on `foreman-deb`.

Reply:
We should extract this into a separate PR done by an engineer as well.